uuid_maker/maker

```diff
@@ -0,0 +1,7 @@
+import basicItemList from "/uuid_maker/basicitems-data_uuid.json";
+import { v4 as uuidv4 } from "uuid";
+
+const newList = basicItemList.map((obj) => ({ ...obj, id: uuidv4() }));
+const jsonList = JSON.stringify(newList);
+console.log(jsonList);
+
```

Comment:
perfect little script for adding uuids. You can use this script to add `slug: slugify(object.name)` as well.

make sure not to commit one-off scripts to your codebase.